server/utils/match/index.test.ts

```diff
@@ -330,6 +331,7 @@ describe('matchUtils', () => {
         apType,
         startDate,
         durationDays,
+        criteria,
       })
 
       expect(result).toEqual(
```

Comment:
This assertion is a bit close to the implementation itself. Could the assert be a bit more explicit, by hradcoding the expectations?